.github/workflows/build-storybook-ui.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           vercel-project-id: ${{ secrets.STORYBOOK_UI_VERCEL_PROJECT_ID }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
           scope: ${{ secrets.VERCEL_ORG_ID }}
-          alias-domains: ${{ env.BRANCH_PREVIEW_SUBDOMAIN }}.dagit-storybook.dagster-docs.io
+          alias-domains: ${{ env.BRANCH_PREVIEW_SUBDOMAIN }}.ui-storybook.dagster-docs.io
```

Comment:
Seems to be consistent with new naming scheme the names should be either:

- `core-storybook`
- `components-storybook`

Or:

- `ui-core-storybook`
- `ui-components-storybook`

Just `ui- storybook` is confusing now because that package was renamed, and tehre is also `ui-core`. @hellendag what do you think